##########################
#
# Object Recognition 
#
##########################

##########################
# Object IDs
##########################
std_msgs/String object_name
int32 object_id
int32 confidence
##########################
# Object 3D Position
##########################
geometry_msgs/Point32 position
